src/identitas/proquint.clj

```diff
@@ -15,11 +15,11 @@ equivalent."
 (def ^:private int-to-vowel
   '[\a \i \o \u])
 
-(def ^:private mask-4 0xF0000000)
-(def ^:private mask-2 0xC0000000)
+(def ^:private mask-4 (unchecked-int 0xF0000000))
```

Comment:
Unhandled clojure.lang.Compiler$CompilerException
   Error compiling proquint.clj at (18:25)

             Compiler.java: 1699  clojure.lang.Compiler$StaticMethodExpr/eval
             Compiler.java:  439  clojure.lang.Compiler$DefExpr/eval
             Compiler.java: 6787  clojure.lang.Compiler/eval
             Compiler.java: 7227  clojure.lang.Compiler/load
                      REPL:    1  user/eval10296
             Compiler.java: 6782  clojure.lang.Compiler/eval
             Compiler.java: 6745  clojure.lang.Compiler/eval
                  core.clj: 3081  clojure.core/eval
                  main.clj:  240  clojure.main/repl/read-eval-print/fn
                  main.clj:  240  clojure.main/repl/read-eval-print
                  main.clj:  258  clojure.main/repl/fn
                  main.clj:  258  clojure.main/repl
               RestFn.java: 1523  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   87  clojure.tools.nrepl.middleware.interruptible-eval/evaluate/fn
                  AFn.java:  152  clojure.lang.AFn/applyToHelper
                  AFn.java:  144  clojure.lang.AFn/applyTo
                  core.clj:  630  clojure.core/apply
                  core.clj: 1868  clojure.core/with-bindings*
               RestFn.java:  425  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   85  clojure.tools.nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:  222  clojure.tools.nrepl.middleware.interruptible-eval/interruptible-eval/fn/fn
    interruptible_eval.clj:  190  clojure.tools.nrepl.middleware.interruptible-eval/run-next/fn
                  AFn.java:   22  clojure.lang.AFn/run
   ThreadPoolExecutor.java: 1142  java.util.concurrent.ThreadPoolExecutor/runWorker
   ThreadPoolExecutor.java:  617  java.util.concurrent.ThreadPoolExecutor$Worker/run
               Thread.java:  748  java.lang.Thread/run

1. Caused by java.lang.IllegalArgumentException
   Value out of range for int: 4026531840